gradle.properties

```diff
@@ -1 +1 @@
-version=1.15.0-SNAPSHOT
+version=1.14.2-SNAPSHOT
```

Comment:
Was there a reason for this version change? I presume you're rebased off a branch and picked up an old version number?